pkg/time/time_skew_amd64.go

```diff
@@ -154,3 +154,140 @@ func (it *TimeSkew) Recover(pid int) error {
 	zeroSkew := NewTimeSkewWithCustomFakeImage(0, 0, it.clockIDsMask, it.fakeImage)
 	return zeroSkew.Inject(pid)
 }
+
+// ============gettimeofday injector================
```

Comment:
Please remove this comment.

If you think the rest of the codes should be splited, please consider creating another file. :D